### PR TITLE
RIA-3084 Legal rep submits appeal with help with fees

### PR DESCRIPTION
**Pull request by tarun-palisetty**
### JIRA link (if applicable) ###
[RIA-3084](https://tools.hmcts.net/jira/browse/RIA-3084)

### Change description ###
Legal rep submits appeal with help with fees

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/functionalTest/resources/scenarios/RIA-3084-submit-appeal-with-help-with-fees.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+  "description": "RIA-3686 Legal representative submits appeal with help with fees",
```
Comment (divisathyan) [inline src/functionalTest/resources/scenarios/RIA-3084-submit-appeal-with-help-with-fees.json:2]: Change the RIA number to 3084

**Review by divisathyan — DISMISSED** [review]
Small correction in functional test. The commit for RIA-3084 looks good

**Review by divisathyan — APPROVED** [review]
3084 commit changes are good

**Review by divisathyan — APPROVED** [review]
Rebase change.

**Review by makowalski — APPROVED** [review]
LGTM

**Pull request merged**